NRF: align to nrfx 4.0

masz-nordic opened the pull request:
This is a hotfix to align with nrfx 4.0.
MDK paths have changed.

A proper rework should be done to align fully with nrfx and remove incompatibilities nrf HW models and nrfx.

masz-nordic committed:
NRF: align to nrfx 4.0

This is a hotfix to align with nrfx 4.0.
MDK paths have changed.

A proper rework should be done to align fully with nrfx and
remove incompatibilities nrf HW models and nrfx.

Signed-off-by: Marcin Szymczyk <marcin.szymczyk@nordicsemi.no>

aescolar commented:
@masz-nordic this repo is just an automated replica. Could you please send the PR to https://github.com/BabbleSim/ext_NRF_hw_models/ ?

masz-nordic commented:
Sure, just wanted a quick revision for an upstream zephyr PR. When it's confirmed to fix all issues I'll make one to the other repo.

nika-nordic committed:
GPIOTE: Enable nRF5340 Network instance

It is required for Zephyr GPIO driver aligned to nrfx 4.0.

Signed-off-by: Nikodem Kastelik <nikodem.kastelik@nordicsemi.no>

nika-nordic committed:
NRF: add option to skip including mocked MDK symbols

Some software components may depent on hardware properties
described by MDK. Allow to access them rather than mocked
counterparts in such cases.

Signed-off-by: Nikodem Kastelik <nikodem.kastelik@nordicsemi.no>
